Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```
@@ -5043,8 +5043,10 @@ MediaSourcePrefersDecompressionSession:
     WebKitLegacy:
       default: false
     WebKit:
+      "CPU(X86_64) || CPU(X86)": false
       default: true
     WebCore:
+      "CPU(X86_64) || CPU(X86)": false
```

Why not do something like?
```suggestion
      default: WebKit::defaultMediaSourcePrefersDecompressionSession()
WebCore:
      default: WebKit::defaultMediaSourcePrefersDecompressionSession()
```

where

```
bool defaultMediaSourcePrefersDecompressionSession()
{
#if CPU(X86_64) || CPU(X86)
    return false;
#else
    return true;
#endif
}
```

---

(as a workaround, because I do think your initial attempt makes sense)

---

> (as a workaround, because I do think your initial attempt makes sense)

it's surprising that it gives some static assert error, because a few line below we have exactly the same default being true, and false on other. yet those cause no issue and with a "stable" one.